### ISSUE TITLE
Many Windows OS dependency

### ISSUE THREAD

**Issue by drmn68** [Schloartz/Phoenix#1]
Never be able to launch in MacOS :-(

**Comment by johannesschweig** [comment]
Yeah, it is true, I have not really aimed for platform-independency. But I think you should be able to leave the Intellitype out (that is just for hotkey functionality). I am not a very proficient java programmer, so any advice is highly appreciated.
